synapse/storage/database.py

```diff
@@ -267,6 +267,17 @@ def execute_batch(self, sql: str, args: Iterable[Iterable[Any]]) -> None:
             for val in args:
                 self.execute(sql, val)
 
+    def execute_values(self, sql: str, *args: Any, **kwargs) -> None:
+        """Corresponds to psycopg2.extras.execute_values. Only available when
+        using postgres.
+        """
+        assert isinstance(self.database_engine, PostgresEngine)
+        from psycopg2.extras import execute_values  # type: ignore
+
+        return self._do_execute(
```

Comment:
Any reason for this return statement if it returns `None`?

Reply:
Oh, woops, it actually does return something :facepalm:

Reply:
Have fixed this up